app/templates/_instructions.html

```diff
@@ -0,0 +1,174 @@
+{% macro r(dataset) -%}
+<p>In order to use Data Package in R follow instructions below:</p>
+<div class="highlight">
+<pre>
+<span></span>install.packages<span class="p">(</span><span class="s">"devtools"</span><span class="p">)</span>
+<span class="kn">library</span><span class="p">(</span>devtools<span class="p">)</span>
+install_github<span class="p">(</span><span class="s">"hadley/readr"</span><span class="p">)</span>
+install_github<span class="p">(</span><span class="s">"ropenscilabs/jsonvalidate"</span><span class="p">)</span>
+install_github<span class="p">(</span><span class="s">"ropenscilabs/datapkg"</span><span class="p">)</span>
+
+<span class="c1">#Load client</span>
+<span class="kn">library</span><span class="p">(</span>datapkg<span class="p">)</span>
+
+<span class="c1">#Get Data Package</span>
+datapackage <span class="o">&amp;</span>lt<span class="p">;</span>
+<span class="o">-</span> datapkg_read<span class="p">(</span>
+<span class="s">"https://bits.datapackaged.com/metadata/{{ dataset.owner }}/{{ dataset.name }}/_v/latest"</span><span class="p">)</span>
```

Comment:
@zelima in R, it is supposed to be so. See: https://github.com/ropenscilabs/datapkg

But I will fix new line issue, that's definitely not correct.